orga/app/adapters/application.js

```diff
@@ -1,29 +1,22 @@
-import { inject as service } from '@ember/service';
-import { computed } from '@ember/object';
-import DS from 'ember-data';
+import JSONAPIAdapter from '@ember-data/adapter/json-api';
 import DataAdapterMixin from 'ember-simple-auth/mixins/data-adapter-mixin';
 import ENV from 'pix-orga/config/environment';
+import ajaxQueue from '../services/ajax-queue';
 
-export default DS.JSONAPIAdapter.extend(DataAdapterMixin, {
-  host: ENV.APP.API_HOST,
-  namespace: 'api',
+export default class ApplicationAdapter extends JSONAPIAdapter.extend(DataAdapterMixin) {
+  host = ENV.APP.API_HOST;
+  namespace = 'api';
 
-  session: service(),
-
-  /*
-  * Appelé uniquement lorsqu'on est connecté
-  * ref: http://ember-simple-auth.com/api/classes/BaseAuthorizer.html#method_authorize
-  */
-  authorize(xhr) {
-    const { access_token } = this.get('session.data.authenticated');
-    xhr.setRequestHeader('Authorization', `Bearer ${access_token}`);
-  },
-
```

Comment:
On n'utilise session en dessous, il faudrait peut être garder l'appel au service.
Pourquoi retirer l'authorize ?

Reply:
C'est la nouvelle manière de faire recommandée par ember-simple-auth depuis la montée en version de la dépendance. Il manquait plus qu'orga pour se mettre à la page, les autres applis c'était déjà bon.
https://github.com/simplabs/ember-simple-auth/blob/master/guides/upgrade-to-v3.md